ftest-pumba/src/test/java/org/arquillian/cube/q/pumba/PumbaFunctionalTestCase.java

```diff
@@ -0,0 +1,49 @@
+package org.arquillian.cube.q.pumba;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.model.Container;
+import org.arquillian.cube.q.api.ContainerChaos;
+import org.arquillian.cube.q.spi.StandaloneContainer;
+import org.hamcrest.CoreMatchers;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+
+@RunWith(Arquillian.class)
+public class PumbaFunctionalTestCase {
+
+    @ArquillianResource
+    ContainerChaos containerChaos;
+
+    @ArquillianResource
+    DockerClient dockerClient;
+
+    @Test
+    public void shouldKillContainers() throws Exception {
+        containerChaos
+                .onCubeDockerHost()
+                    .killRandomly(
+                            ContainerChaos.ContainersType.regularExpression("^pingpong"),
+                            ContainerChaos.IntervalType.interval(4),
+                            ContainerChaos.KillSignal.SIGTERM
+                    )
+                .exec();
+
+
+        TimeUnit.SECONDS.sleep(12);
```

Comment:
But you cannot assume it will take just 12s - if run on the slower machine than yours it will always fail, if on faster it's simply adding more waiting time for the build finish which is a waste :)